monai/transforms/spatial/array.py

```diff
@@ -1064,3 +1065,29 @@ def __call__(self, img, spatial_size=None, mode=None):
             grid[:3] += gaussian(self.rand_offset[None])[0] * self.magnitude
             grid = self.rand_affine_grid(grid=grid)
         return self.resampler(img, grid, mode)
+
+
+class TestIdentity(Transform):
```

Comment:
1. I think this transform is a very useful utility function, could you please move it to `transforms/utility/` instead of spatial transforms folder?
2. Maybe `SimulateDelay` is a more suitable name for this transform?
Thanks.

Reply:
1. Done
2. Done